.github/workflows/release_nightly.yml

```diff
@@ -269,6 +266,19 @@ jobs:
           npm run build
           npm run docs
 
+      - name: Sign Firefox extension
+        id: sign-firefox
+        continue-on-error: true
+        env:
+          FIREFOX_EXTENSION_ID: ${{ secrets.FIREFOX_EXTENSION_ID }}
+          MOZILLA_API_KEY: ${{ secrets.MOZILLA_API_KEY }}
+          MOZILLA_API_SECRET: ${{ secrets.MOZILLA_API_SECRET }}
+        working-directory: web
+        shell: bash -l {0}
+        run: |
+          cd web/packages/extension/
```

Comment:
Instead of doing `cd` manually, you could declare it using `working-directory`.

Reply:
Lots of other steps here, including the very next one, use `cd`, which is why I went for it. They all seem to start in the root of the repo.

Reply:
ah, I see the working-directory here, missed that.